crates/manifest/src/schema/common.rs

```diff
@@ -39,7 +39,7 @@ pub enum ComponentSource {
         /// `package = "example:component"`
         package: PackageRef,
         /// `version = "1.2.3"`
-        version: semver::Version,
+        version: String,
```

Comment:
Another option here would be a custom `deserialize_with`; the thing I appreciate about using `Version` here is that you get nice location information in errors.